parser/config.go

```diff
@@ -30,6 +31,8 @@ Use 'deis help [command]' to learn more.
 		return configPull(argv)
 	case "config:push":
 		return configPush(argv)
+	case "config:routing":
```

Comment:
Yeah, I was confused by the existence of both `config:routing` and `routing:enable`. But if that's just old code no big deal.